src/client/js/partials/breaches.js

```diff
@@ -45,6 +45,7 @@ function handleEvent (e) {
     case e.target.matches('custom-select[name="email-account"]'):
       state.selectedEmail = e.target.value
       breachesTable.querySelectorAll('span[data-email]').forEach(message => message.toggleAttribute('hidden', message.dataset.email !== e.target.value))
+      document.cookie = `monitor.selected-email-index=${Array.from(e.target.shadowRoot.querySelectorAll('option')).findIndex(optionEl => optionEl.value === e.target.value)}`
```

Comment:
I'm not a big fan of using the DOM as the source of truth for data. Another option would be to add the list of email addresses to the state object - let me know if you prefer that.

Reply:
Also, this line is looong. In absence of an automated code formatter, happy to take suggestions; I would've extracted the index into a separate value, but ESLint wouldn't let me do that in `case` statements.

Reply:
Hope you accept contributions!  I added code via https://github.com/mozilla/blurts-server/pull/2867/commits/fc71963f31c97e9b3d136dde638587f32464c770 to expose `selectedIndex` in `<custom-select>`.  (This property is native to a standard `<select>`.)  

This significantly shortens your line 😉

Reply:
That works, thanks!